### PR TITLE
Pylint fixes

### PR DESCRIPTION
**Pull request by mbasti-rh**
I updated pylint fixes from Jan Barta

### CODE REVIEW

File: ipapython/install/util.py [hunk 1]
```diff
@@ -142,23 +142,23 @@ def __delete__(self, obj):
...
+            assert value is cls
 
-            self.__outer_class__ = cls
-            self.__outer_name__ = name
-            self.__name__ = '.'.join((cls.__name__, name))
-            self.__qualname__ = self.__name__
+            cls.__outer_class__ = outer_class
+            cls.__outer_name__ = name
+            cls.__name__ = '.'.join((outer_class.__name__, name))
+            cls.__qualname__ = outer_class.__name__
```
Comment (tomaskrizek) [inline ipapython/install/util.py]: this should be `cls.__qualname__ = cls.__name__`

**Comment by tomaskrizek** [comment]
NACK, please see inline comments

File: ipaserver/plugins/dogtag.py [hunk 2]
```diff
@@ -1750,12 +1750,7 @@ def revoke_certificate(self, serial_number, revocation_reason=0):
         # Return command result
         cmd_result = {}
 
-        if parse_result.get('revoked') == 'yes':
-            cmd_result['revoked'] = True
-        else:
-            cmd_result['revoked'] = False
-
-        return cmd_result
```
Comment (mbasti-rh) [inline ipaserver/plugins/dogtag.py]: Nice catch

File: ipaserver/plugins/dogtag.py [hunk 3]
```diff
@@ -1750,12 +1750,9 @@ def revoke_certificate(self, serial_number, revocation_reason=0):
...
         cmd_result = {}
 
-        if parse_result.get('revoked') == 'yes':
-            cmd_result['revoked'] = True
-        else:
-            cmd_result['revoked'] = False
+        cmd_result['revoked'] = parse_result.get('revoked') == 'yes'
 
-        return cmd_result
+        return  cmd_result
```
Comment (tomaskrizek) [inline ipaserver/plugins/dogtag.py]: PEP8 violation

**Comment by tomaskrizek** [comment]
Please fix the typo.

**Comment by mbasti-rh** [comment]
I had to rebase, there were conflicts due the latest pushed patches

**Comment by mbasti-rh** [comment]
Fixed upstream
master:
https://fedorahosted.org/freeipa/changeset/36484e8672f5ee1fdc2bd57622e330ab8dbb7671
https://fedorahosted.org/freeipa/changeset/275e85d076607ce317b3aeca467167fac55bf396
https://fedorahosted.org/freeipa/changeset/cdecbcd0a175e010057beae6f7fb74fd67856ca1
https://fedorahosted.org/freeipa/changeset/568f9da331af14e5f05764c46f51a0410da1e49c
https://fedorahosted.org/freeipa/changeset/9bc57a01e1c0942e1a94ac0d948c8c5f8c0d4dcc
https://fedorahosted.org/freeipa/changeset/f252f50987cfb1234671ca1742c11a0eebe8633c
https://fedorahosted.org/freeipa/changeset/8420d04f383b958660934ccf3c7c3bf9b27ac30c
https://fedorahosted.org/freeipa/changeset/71b3352ad0e0aa105c90e490a41645dfcc46ce87

**Pull request closed without merging**